Add View helpers

jailln opened the pull request:
Add some helper functions in `View` to get threejs renderer, renderer size and camera to avoid having to know itowns internal structure to access them.

jailln committed:
feat(View): add getters for threejs renderer and camera

Use `View.renderer` to get the threejs renderer of the view
Use `View.camera3D` to get the threejs camera of the view

ftoromanoff commented on examples/laz_dragndrop.html:
```diff
@@ -45,23 +45,21 @@
                         geometry.boundingBox.getCenter(lookAt);
 
-                        camera.far = Math.max(2.0 * size.length(), camera.far);
+                        cameraThree.far = Math.max(2.0 * size.length(), camera.far);
```
var cameraThree= view.getThreeJsCamera();
This line seems to be  missing ?

jailln replied:
well spoted! done

ftoromanoff commented on src/Core/Prefab/GlobeView.js:
```diff
@@ -99,8 +99,8 @@ class GlobeView extends View {
 
-        this.camera.camera3D.near = Math.max(15.0, 0.000002352 * ellipsoidSizes.x);
-        this.camera.camera3D.far = ellipsoidSizes.x * 10;
+        this.getThreeJsCamera().near = Math.max(15.0, 0.000002352 * ellipsoidSizes.x);
```
Maybe add a line ``` const camera3D = this.getThreeJsCamera(); ``` ?

jailln replied:
ok, done

ftoromanoff commented:
By adding helper functions to get specific variable, should we not modify these variables to set them as private ?
We can also instead of implementing function to get variables, use setter and getter ?
I don't really know what is the best way to work around these from a architectural/api point of view...

ftoromanoff reviewed (commented):
I get the point that private variable in JS might not be well handled.

For the second point: using methode getVariable() or using setter/getter I can't say what is the best usage, we might need to discuss that internaly.

jailln commented:
I think setting them as private could be done but it is not related to this PR. In addition, I'm not fond of private fields implementation in JS ([side effects](https://lea.verou.me/blog/2023/04/private-fields-considered-harmful/), badly interpreted by dev tools). But this is another subject :)

Regarding getter vs the methods I implemented, in my opinion both are valid and it's only a matter of coding style and API. I don't have a strong opinion on which one we should used, as long as it's decided quickly. Please let me know, so we can move on quickly.

ftoromanoff commented:
> I think setting them as private could be done but it is not related to this PR. In addition, I'm not fond of private fields implementation in JS ([side effects](https://lea.verou.me/blog/2023/04/private-fields-considered-harmful/), badly interpreted by dev tools). But this is another subject :)
> 
> Regarding getter vs the methods I implemented, in my opinion both are valid and it's only a matter of coding style and API. I don't have a strong opinion on which one we should used, as long as it's decided quickly. Please let me know, so we can move on quickly.

After discussion, I think we should opt for the setter/getter methods. 
In the case of camera being already in use (by the itowns.camera) we might use ```get camera3D``` or any other name and add a deprecation warning to get camera (wich should be internaly rename _camera.

jailln commented:
Sorry I don't understand what you mean by:
> In the case of camera being already in use (by the itowns.camera) we might use `get camera3D` or any other name and add a deprecation warning to get camera (wich should be internaly rename _camera.

Can you give more details please ?

jailln committed:
doc(View): document View properties

jailln commented:
I changed to implemented to getters, removed the `getRendererSize` because one can use `View.renderer` and then gets the size if one needs it. I also added a commit to document `View` properties (including `camera3D` and `renderer`).

Let me know if that's what you meant by your last comment.
